packages/peregrine/lib/talons/SignIn/__tests__/useSignIn.spec.js

```diff
@@ -131,6 +131,7 @@ test('returns correct shape', () => {
             "shouldRender": false,
```

Comment:
Hi @RaghavendraTirumalasetti - Would you sign the Adobe CLA? [Details](https://github.com/magento/pwa-studio/pull/4038/checks?check_run_id=11466390104)

Please fill out Description & remove TODO's for the PR description [Details](https://github.com/magento/pwa-studio/pull/4038#issuecomment-1436958883)

Afterwards, please rerun builds & they should pass 👍

Reply:
Code looks good, thank you